api/node/Dockerfile.open

```diff
@@ -0,0 +1,22 @@
+FROM image-registry.openshift-image-registry.svc:5000/9b301c-tools/node:18-bullseye AS build
```

Comment:
Need to pull the node image from Openshift because docker blocks the Government after they have requested too many images.